code/modules/mob/living/carbon/human/species.dm

```diff
@@ -921,6 +924,38 @@
 		else
 			H.throw_alert("nutrition", /obj/screen/alert/starving)
 
+/datum/species/proc/handle_disgust(mob/living/carbon/human/H, var/metabolism_rate)
```

Comment:
actually on second thoughts, this should live in the organ, not on the species or as a status effect

move it there instead